frontend/common/src/lang/fr.json

```diff
@@ -1935,5 +1935,9 @@
   "tW4k56": {
     "defaultMessage": "Un champ obligatoire est vide : Anglais – Votre travail",
     "description": "Error message that Work Tasks in English must be filled"
+  },
+  "w2tWfH": {
+    "defaultMessage": "Un champ obligatoire est manquant : Titres des volets/emplois",
```

Comment:
This translation does not fit with what has been translated before. There are several other strings that follow this format for _You are missing a required field:_ but use **vide** instead of **manquant** for _missing_ 😕

![Screen Shot 2022-11-02 at 13 55 47](https://user-images.githubusercontent.com/3046459/199566025-1ee794e3-5b42-4dd8-a829-71ae4c201df4.png)

Reply:
I think this might be different because I had to ask for it separately since it was lost.